packages/screens/RiotGame/component/RipperGridSelectorModal.tsx

```diff
@@ -170,9 +170,8 @@ export const RipperSelectorModal: React.FC<RipperSelectorModalProps> = ({
                           activeOpacity={0.7}
                           onPress={() => selectRipper(item)}
                         >
-                          <LegacyTertiaryBox
-                            noBrokenCorners
-                            mainContainerStyle={[
+                          <TertiaryBox
+                            style={[
```

Comment:
Here you can directly write `style={{blabla}}`

Reply:
Done.